python/GangaLHCb/test/GPI/TestDataFiles.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+
+from Ganga.testlib import external
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+
+
+class TestDataFiles(GangaUnitTest):
+
+    @external
+    def testDataFiles(self):
+
+        from Ganga.GPI import DiracFile, config
+
+        print config['LHCb']
+        # LFNs
+        name = 'test.txt'
+        lfn = DiracFile(lfn=name)
+        assert lfn.lfn == name
+        lfn = DiracFile('lfn:'+name)
+        assert lfn.lfn == name
+        lfn = DiracFile('LFN:'+name)
+        assert lfn.lfn == name
+        try:
+            lfn = DiracFile('pfn:'+name)
```

Comment:
A warning is printed and a `LocalFile` created so it does work. However, if it's a valid test is up for debate as I just copied it wholesale from the legacy tests.